src/common/config_parser.cpp

```diff
@@ -324,13 +324,23 @@ void ConfigParser::addOptionsModel(cli::CLIWrapper& cli) {
         "Dropout for transformer attention (0 = no dropout)");
     cli.add<float>("--transformer-dropout-ffn",
         "Dropout for transformer filter (0 = no dropout)");
+
   }
   cli.switchGroup(previous_group);
   // clang-format on
 }
 
 void ConfigParser::addOptionsTraining(cli::CLIWrapper& cli) {
-  auto previous_group = cli.switchGroup("Training options");
+  auto previous_group = cli.switchGroup("Signal Handling");
```

Comment:
@snukky can you comment if this is the right place to put the handler or if you would like it be located somewhere else? @ugermann is signal handling currently only specific to training or are the other things like translation and scoring reacting as well?

Reply:
Currently it's only specific to training. The other option where it might be useful is a translation server, where we want to might shut down a server for load balancing / migration to a different node but want to complete pending requests before the particular server instance goes offline.

Reply:
Right, it seems to be that signal handling is a little bit similar to things like `--log-level` or even `--help`. Sort of a meta-option.  Let's @snukky have the last word here.

Reply:
If potentially the option can be used outside the training too, I would add it under 'General options'; only for the trainer. Creating a separate section for a single option will also make the help even more longer.

Reply:
Now in 'General options' as suggested, without separate section.